### PR TITLE
[ticket/15534] Update extensions database link in ACP for phpBB 3.2

### PR DESCRIPTION
**Pull request by rxu**
For 3.2.x only as for the master branch this is not the case.

Checklist:

- [x] Correct branch: master for new features; 3.2.x for fixes
- [x] Tests pass
- [x] Code follows coding guidelines: [master](https://area51.phpbb.com/docs/dev/master/development/coding_guidelines.html) and [3.2.x](https://area51.phpbb.com/docs/dev/3.2.x/development/coding_guidelines.html)
- [x] Commit follows commit message [format](https://area51.phpbb.com/docs/dev/3.2.x/development/git.html)

Tracker ticket (set the ticket ID to **your ticket ID**):

<a href="https://tracker.phpbb.com/browse/PHPBB3-15534">PHPBB3-15534</a>.

### CODE REVIEW

**Comment by rxu** [comment]
!set Styles

File: phpBB/adm/style/acp_ext_list.html [hunk 1]
```diff
@@ -7,7 +7,7 @@ <h1>{L_EXTENSIONS_ADMIN}</h1>
 	<p>{L_EXTENSIONS_EXPLAIN}</p>
 
 	<fieldset class="quick">
-		<span class="small"><a href="https://www.phpbb.com/go/customise/extensions/3.1" target="_blank">{L_BROWSE_EXTENSIONS_DATABASE}</a> &bull; <a href="{U_VERSIONCHECK_FORCE}">{L_VERSIONCHECK_FORCE_UPDATE_ALL}</a> &bull; <a href="javascript:phpbb.toggleDisplay('version_check_settings');">{L_SETTINGS}</a></span>
+		<span class="small"><a href="https://www.phpbb.com/customise/db/extensions-36/3.2" target="_blank">{L_BROWSE_EXTENSIONS_DATABASE}</a> &bull; <a href="{U_VERSIONCHECK_FORCE}">{L_VERSIONCHECK_FORCE_UPDATE_ALL}</a> &bull; <a href="javascript:phpbb.toggleDisplay('version_check_settings');">{L_SETTINGS}</a></span>
```
Comment (marc1706) [inline phpBB/adm/style/acp_ext_list.html]: Please change it to `https://www.phpbb.com/go/customise/extensions/3.2`
That link will be available in a day or two. ;)

**Comment by DavidIQ** [comment]
Maybe it should be more dynamic and set the version based on the installation's version. Otherwise this will need updating for each new major release. Unless we're getting rid of this link altogether after 3.2? (There's an extension installer there...)

**Comment by marc1706** [comment]
For that we'd need to use the branch (rather easy as well). But as we probably might revamp that page altogether I think it's better to just update this now.

Can you also update the links to the styles database in acp_styles.html?
Link should be:
`https://www.phpbb.com/customise/styles/3.2`

**Comment by rxu** [comment]
@marc1706 Done.

**Comment by vinny** [comment]
[adm/style/acp_language.html#L72](https://github.com/phpbb/phpbb/blob/3.2.x/phpBB/adm/style/acp_language.html#L72) should be updated too.

**Comment by marc1706** [comment]
Fixing those links as well. Just replace 3.1 with 3.2 ;)

**Comment by rxu** [comment]
Well, if you believe those links will actually work soon, then here you are 😆

**Comment by marc1706** [comment]
Those links will work once we push 3.2.3-RC1 ;)

**Pull request merged**